src/lib/utils.ts

```diff
@@ -259,14 +260,10 @@ export function encodeKey(
         const dataElement = value[index];
         if (index === 0) {
           // This is arrayLength as the first element in the raw array
+          // encoded as a uint128
           results.push({
             key: schema.key,
-            value: encodeKeyValue(
-              'Number',
```

Comment:
So, you're not using `ArrayLength` type here and decided to move to a simpler approach?

Reply:
No I did not use this because this valueContent is not defined in LSP2.

I guess we might want to deprecate it.

See this discussion comments: https://github.com/ERC725Alliance/erc725.js/pull/287#discussion_r1193203269